### ISSUE TITLE
dota 2 graphical glitch

### ISSUE THREAD

**Issue by Dota2player** [ValveSoftware/Dota-2#1845]
![screenshot from 2015-09-04 20 20 09](https://cloud.githubusercontent.com/assets/13817845/9692667/7a987046-5342-11e5-99b0-1d7b90d083af.png)
Hello

this is what i mean and this happens alot and soemtimes when i move my mouse it shows red or black squares, it also happens for alot of other things as you can see such as a mega kill(it will show a black box like it does in the image) Please help this is really annoying

**Comment by dpi** [comment]
Ive been getting this since around Friday (4 Sept 2015).

Menu UI element background flickers. On occasion the in game background which stretches from left to right, behind hero portrait, will flicker black. Leaving only portrait images visible.

Linux 64 / Elementary (Ubuntu derivative)

**Comment by Dota2player** [comment]
Yes mate that is exactly my issue. I think it is to do with the dots 2 reborn box as a fterca match it shows it automatically and then the issue starts

**Comment by Dota2player** [comment]
I have Linux 14.04.03 but I think this is a global issue on Linux that is caused by the dota 2 reborn box in the menu

**Comment by MaikuMori** [comment]
I can confirm having this issue as well on normal Dota 2 client recently. My intuition says that it has something to do with alt-tabing, at least that seems to trigger it for me. But I haven't managed to pinpoint exact sequence of actions to trigger it.

Not sure if it happens in reborn client since I don't get to play with it much due to it still being a bit too buggy.
